examples/out-of-tree/kube-apiserver.yaml

```diff
@@ -10,7 +10,7 @@ spec:
   hostNetwork: true
   containers:
     - name: kube-apiserver
-      image: k8s.gcr.io/hyperkube-amd64:v1.16.0-beta.1
+      image: k8s.gcr.io/hyperkube-amd64:v1.16.0
```

Comment:
Thanks for the update.

since azure-cloud-node-manager:v0.4.1 only works with v1.17, could you replace the hyperkube tag with v1.17.2?

Reply:
ping @ialidzhikov